Gamma_Analysis.py

```diff
@@ -0,0 +1,339 @@
+"""
+This will perform a gamma analysis
+It will collect two spectra:
+M = Measurement Spectrum
+B = Background Spectrum
+"""
+from __future__ import print_function
+from SpectrumFileBase import SpectrumFileBase
+import SPEFile
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import os
+# exec('Isotope_identification.py')
+# Use bottom line for Python 3 only. Use top comment for Python 2.7 only.
+exec(open('Isotope_identification.py').read())
+
+
+EFFICIENCY_CAL_COEFFS = [5.1164, 161.65, 3952.3, 30908]
+
+
+def absolute_efficiency(energy, coeffs=EFFICIENCY_CAL_COEFFS):
+    """
+    Returns absolute efficiencies for a given set of energies, based on a
+    provided efficiency calibration.
+    """
+    efficiency = []
+    for i in range(len(energy)):
+        efficiency.append(math.exp(coeffs[3] *
+                          (math.log(energy[i])/energy[i])**3 -
+                          coeffs[2]*(math.log(energy[i])/energy[i])**2 +
+                          coeffs[1]*(math.log(energy[i])/energy[i]) -
+                          coeffs[0]))
```

Comment:
Have you checked this equation? Calculate one efficiency using your calculator, and make sure you get the same answer.